projects/core/bindaas-core-impl/src/main/java/edu/emory/cci/bindaas/core/rest/security/SecurityHandler.java

```diff
@@ -85,6 +86,7 @@ public static void invalidateJWT(String jws) {
 	public final static String API_KEY = "api_key";
 	public final static String JWT = "jwt";
 	public final static String AUTH_HEADER = "Authorization";
+	public final static String ROLE = "role";
```

Comment:
Instead of defining constants such as "role" and "api_key" several times over multiple classes, why not unify them as public constants in a Constants class?

I think there is already a constant class in bindaas/core. If so, just use that class. Otherwise, you can create one and move these constants there.

That will help us edit the constants all at once (Isn't it the motivation behind creating final static String constants anyway?)

Reply:
Makes sense, will do.